htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -1200,8 +1200,12 @@ public function update($user = null, $notrigger = 0)
 		if (empty($this->total_tva)) {
 			$this->total_tva = 0;
 		}
-		//	if (isset($this->total_localtax1)) $this->total_localtax1=trim($this->total_localtax1);
-		//	if (isset($this->total_localtax2)) $this->total_localtax2=trim($this->total_localtax2);
+		if (isset($this->total_localtax1)) {
+			$this->total_localtax1=trim($this->total_localtax1);
```

Comment:
Welcome back juanjo !
We can't use trim here, because total_localta1 should be a float here.  And trim is designed for string.

Reply:
Thank you Laurent for the welcome,
Ok, I had only uncommented the lines, now changed by empty check and make and also done for total_ttc